### PR TITLE
Do not lock `Mutex` to get the network

### PR DESCRIPTION
**Pull request by luckysori**

### CODE REVIEW

File: crates/ln-dlc-node/src/ln_dlc_wallet.rs [hunk 1]
```diff
@@ -123,9 +127,7 @@ impl Blockchain for LnDlcWallet {
 
     #[autometrics]
     fn get_network(&self) -> Result<Network, Error> {
-        self.ln_wallet
-            .network()
-            .map_err(|e| Error::BlockchainError(e.to_string()))
+        Ok(self.network)
```
Comment (holzeis) [inline crates/ln-dlc-node/src/ln_dlc_wallet.rs:130]: nit: This is now infallible, we could get rid of the result.

**Review by holzeis — APPROVED** [review]

File: crates/ln-dlc-node/src/ln_dlc_wallet.rs (hunk 1 above)
Comment (luckysori) [reply crates/ln-dlc-node/src/ln_dlc_wallet.rs:130]: You're right! I thought it was a trait method 🤦

Comment (luckysori) [reply crates/ln-dlc-node/src/ln_dlc_wallet.rs:130]: Ah, nvm. It _is_ a foreign trait method.

**Review by klochowicz — APPROVED** [review]
thank you for addressing the TODO ❤️

**Review by klochowicz — APPROVED** [review]
thank you for addressing the TODO ❤️

**Pull request merged**